app/tests/e2e/nightwatchGlobals.js

```diff
@@ -305,40 +272,7 @@ async function switchToAccount(
     },
     [{ address, network, wallet, name }]
   )
-  await browser.refresh()
+  // await browser.refresh() // not needed as getAccountBalance moves to portfolio
   await getAccountBalance(browser)
-  // wait until on portfolio page to make sure future tests have the same state
-  browser.expect.element(".balance-header").to.be.visible.before(10000)
-  // switching to homepage
-  await browser.url(browser.launch_url)
   return true
 }
-
-async function apiUp(browser) {
-  console.log("Testing if API is up")
-  const start = new Date().getTime()
-  // we need to wait until the testnet is up and the account has money
-  let apiUp = false
-  while (!apiUp) {
-    if (new Date().getTime() - start > 90000) {
-      throw new Error("Timed out waiting for API to be up.")
-    }
-    try {
-      // test if the api can return networks list
-      const response = await axios.post(browser.globals.apiURI, {
-        query: `{networks {id}}`,
-      })
-      if (response.data.errors) {
-        throw new Error(JSON.stringify(response.data.errors))
-      }
-      if (Number(response.data.data.networks.length) === 0) {
-        continue
-      }
-      apiUp = true
-    } catch (err) {
-      console.log("Failed to check API", err)
-      console.log("Waiting 1s for API to be up")
-      await new Promise((resolve) => setTimeout(resolve, 1000))
-    }
-  }
-}
```

Comment:
Why do we remove this check? Well, if API is not up it fails anyway no? But will we know why? This should help us identify the reason no

Reply:
I tried to simplify and speed up. E2E tests use the deployed apis which are always up. This is only used for localhost.

Reply:
Cool, thank you for explanation